test/cli/project/test_project.py

```diff
@@ -341,26 +341,45 @@ def test_import(self, mocker):
                     )
 
     def test_export(self, mocker):
+        export_commands = [
+            [
+                "--output-format",
+                "yolo_v4",
+                "--output-file",
+                "export.zip",
+                "--project-id",
+                "object_detection",
+                "--layout",
+                "split",
+                "--verbose",
+                "--api-key",
+                "toto",
+                "--endpoint",
+                "localhost",
+            ],
+            [
+                "--output-format",
+                "raw",
+                "--output-file",
+                "export.zip",
+                "--project-id",
+                "object_detection",
+                "--layout",
+                "split",
+                "--verbose",
+                "--api-key",
+                "toto",
+                "--endpoint",
+                "localhost",
+            ],
+        ]
+
         runner = CliRunner()
         with runner.isolated_filesystem():
-
-            result = runner.invoke(
-                export_labels,
-                [
-                    "--output-format",
-                    "yolo_v4",
-                    "--output-file",
-                    "export.zip",
-                    "--project-id",
-                    "object_detection",
-                    "--layout",
-                    "split",
-                    "--verbose",
-                    "--api-key",
-                    "toto",
-                    "--endpoint",
-                    "localhost",
-                ],
-            )
-            debug_subprocess_pytest(result)
-            assert result.output.count("export.zip")
+            for export_command in export_commands:
```

Comment:
you can use `@pytest.mark.parametrize` to create separate tests (see in other places)

Reply:
good remark! Many other tests need this also.

Reply:
Interesting... When using this decorator on the unittests, it didn't work. I had to use `@parameterized.expand`